web/src/views/HomepageView.jsx

```diff
@@ -12,13 +12,12 @@ const contentStyle = {
 };
 
 export default function HomepageView() {
-   
     const location = useLocation();
     useEffect(() => {
       if (location.state && location.state.message) {
         message.info(location.state.message);
       }
-    }, [location]);
+    })
```

Comment:
I think you removed this because VSCode suggested you to do so. But I guess we need this in order to show messages on page changes.

Reply:
Fixed with the latest commit.